entries/entry05.md

```diff
@@ -1,8 +1,62 @@
 # Entry 5
-##### X/X/XX
+##### 3/15/20
+
+### What I am using to learn    
+So far I have watched a ton of Blender videos to help me with my 3D models for the web game my partner and I are making. Currently, I am learning as I go instead of watching beginner level videos that teach you the basics. For example, if I am stuck on a specific problem on a model I am trying to create, I would search up the problem and solve it as I go.
```

Comment:
Which videos? Link them using MD syntax! Also:
> ... if I am stuck on a specific problem on a model I am trying to create, I would search up the problem and solve it as I go

What problem? What do you search? Provide details!